ui/search.html

```diff
@@ -23,79 +23,111 @@
         <div data-widget="topnavigation"><!-- --></div>
 
         <!-- BODY -->
-        <main class="oae-main-content" role="main">
-
-            <!-- CLIPS -->
-            <div class="oae-clip-container">
-                <div id="search-clip-container">
-                    <div class="oae-clip">
-                        <div class="oae-clip-fold">
-                            <div class="oae-clip-fold-inner"></div>
-                            <div class="oae-clip-fold-outer"></div>
-                        </div>
-                        <div class="clearfix oae-clip-content">
-                            <form action="." id="search-form" class="form-search form-horizontal" role="search" aria-label="__MSG__SEARCH__">
-                                <div class="form-group">
-                                    <input type="text" id="search-query" class="form-control search-query form-control pull-left" title="__MSG__SEARCH_FOR_CONTENT_FOLDERS_PEOPLE_GROUPS__" autofocus />
-                                    <button type="submit" id="search-search-icon" class="btn btn-link pull-left" title="__MSG__SEARCH_FOR_CONTENT_FOLDERS_PEOPLE_GROUPS__">
-                                        <i class="fa fa-search"><span class="sr-only">__MSG__SEARCH__</span></i>
-                                    </button>
-                                </div>
-                            </form>
-                        </div>
+        <main role="main">
+
+            <div class="oae-main-action">
+                <form action="." id="search-form" class="form-search form-horizontal" role="search" aria-label="__MSG__SEARCH__">
```

Comment:
Actually not. There has to be an `action` attribute for iOS to use the Search keyboard instead of the generic one. (That's what gives you the "Search" key instead of the generic "Go" key.)